kratos/imports/import-last-login.js

```diff
@@ -0,0 +1,61 @@
+import { connection, kratos } from './import-users-config.js'
+
+connection.connect(async (error) => {
+  if (error) throw error
+
+  let allIdentities = []
+
+  console.log('Getting list of Kratos identities...')
+  for (let page = 1; true; page++) {
+    const data = await kratos
+      .adminListIdentities(10, page)
+      .then(({ data }) => data)
+    if (!data.length) break
+    allIdentities = [...allIdentities, ...data]
+  }
+  if (allIdentities) {
```

Comment:
Sorry, you may have taken it from the other script, my fault, but it will actually always evaluate to true, since even an empty array is something.
```
> const a = []
undefined
> if (a) { console.log('if called') }
if called
```
To make a correct check
```suggestion
  if (allIdentities.length > 0) {
```
`if (allIdentities.length)` is also possible.